src/main/resources/application.properties

```diff
@@ -58,6 +58,7 @@ spring.mail.properties.mail.smtp.auth: false
 spring.mail.properties.mail.smtp.starttls.enable: false
 
 event.repository.url:
+event.repository.shallow:
```

Comment:
You can set false value as default to shallow flag.
And also describe about what shallow flag is in configuraiton.md documentation file.

Reply:
OK , I will make the shallow flag as true and will add description about shallow flag in documentation